ocfkube/apps/cilium.py

```diff
@@ -12,7 +12,12 @@
         "integration": "containerd",
         "socketPath": "/run/containerd/containerd.sock",
     },
-    "hubble": {"tls": {"auto": {"method": "cronJob"}}},
+    "hubble": {
+        "tls": {"auto": {"method": "cronJob"}},
+        "listenAddress": ":4244",
+        "relay": {"enabled": True},
+        "ui": {"enabled": True},
```

Comment:
Only concern(?) is that we aren't exposing this service anywhere, so we have to port-forward to look at hubble...

Maybe we want to make it accessible from inside our cidr or something?